.mergify.yml

```diff
@@ -48,3 +50,13 @@ pull_request_rules:
       assign:
         add_users:
           - "{{author}}"
+  - name: notify the backport has not been merged yet
+    conditions:
+      - -merged
+      - -closed
+      - author=mergify[bot]
+      - check-success=CLA
+    actions:
+      comment:
+        message: |
+          This pull request has not been merged yet. Could you please review it @{{assignee}}? 🙏
```

Comment:
Nit: Should this says `review and merge it`?

Reply:
Good catch, this was only for testing purposes :phrew! And the one that will benefit from uses it fine:
- https://github.com/elastic/e2e-testing/blob/aa56b1f99d6da71f454a16d0d8dd2ae606d43be9/.mergify.yml#L100-L111

I'll fix it anyway to avoid C&P issues